.github/workflows/main.yml

```diff
@@ -29,6 +29,17 @@ jobs:
         uses: DeLaGuardo/setup-clojure@9.5
         with:
           cli: 1.11.1.1155
+          
+      - name: clj-kondo checks
+        uses: DeLaGuardo/clojure-lint-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why you need token here?

Reply:
> secrets.GITHUB_TOKEN is needed here
> to publish annotations back to github

Reply:
I'll use another action option which doesn't require it